dist/maven-antrun/build-parallel-worlds.xml

```diff
@@ -131,7 +131,7 @@
 
         <!-- Remove the explicily unshimmed files from the common directory -->
         <delete>
-            <fileset dir="${project.build.directory}/parallel-world/spark3xx-common"
+            <fileset dir="${project.build.directory}/parallel-world/spark-shared"
```

Comment:
There are a lot of references to spark3xx-common in the code base, not sure all of them have been updated to the new name, e.g.: coverage-report, shims.md, spark-premerge-build.sh, etc.  Don't we need to update those as well?

Reply:
Thanks for pointing that out I will create a follow-on PR

Reply:
https://github.com/NVIDIA/spark-rapids/pull/11066